src/lm_buddy/jobs/_entrypoints/prometheus.py

```diff
@@ -0,0 +1,131 @@
+# lm-buddy entrypoint to run evaluations using a Prometheus inference server
+# see https://github.com/kaistAI/prometheus/blob/main/evaluation/benchmark/run_absolute_scoring.py
+
+from lm_buddy.jobs.configs import PrometheusJobConfig
+from lm_buddy.integrations.huggingface import HuggingFaceAssetLoader
+from lm_buddy.integrations.wandb import (
+    ArtifactType, 
+    ArtifactLoader, 
+    build_file_artifact, 
+    wandb_init_from_config,
+)
+from fastchat.conversation import get_conv_template
+from transformers import AutoTokenizer
+from openai import OpenAIError, OpenAI
+
+from tqdm import tqdm
+import os
+import json
+import copy
+
+class BadResponseException(Exception):
```

Comment:
what does the bad response mean in this case, is it an error from the server? we might want to specify the exception type a bit more here

Reply:
So, this is more like a Prometheus quirk. The server can return errors (`OpenAIError`) which are trapped out of this class. But sometimes, even if the server response arrives it might be malformed - e.g. when `max_tokens` is too short for Prometheus to return a full answer (so you end up without a score, hence the split error) or when the values for the score are not valid... Which reminds me this has to be a parameter if we want to customize our rubrics! Good catch, thanks for raising this :-)